oval_graph/oval_node.py

```diff
@@ -1,17 +1,11 @@
 '''
-    Modules form my lib and for create ID
+    This file contains a oval node class
```

Comment:
Do you really think that this will be helpful for anyone? We can see that there is OvalNode class in this file. You can describe it more.

Reply:
Or, conversely, one doesn't have to describe it at all. The problem with documentation is that it may not get updated as frequently as the code, so it may become misleading.